controllers/imagerepository_controller.go

```diff
@@ -46,6 +49,12 @@ import (
 	imagev1alpha1 "github.com/fluxcd/image-reflector-controller/api/v1alpha1"
 )
 
+const (
+	ClientCert = "certFile"
+	ClientKey  = "keyFile"
+	CACert     = "cacertFile"
```

Comment:
```suggestion
	CACert     = "caFile"
```

To match Helm https://github.com/fluxcd/flux2/blob/main/cmd/flux/create_secret_helm.go#L122

Reply:
Huh, I looked this up in source-controller/internal/helm and somehow transcribed it wrong 😳